plugins/org.komodo.core/src/org/komodo/repository/ObjectImpl.java

```diff
@@ -68,6 +71,87 @@ ObjectImpl accessOuter() {
     }
 
     /**
+     * Implementation specific and not to be included on the interface!
+     *
+     * @param uow transaction for this operation. Can be null.
+     *
+     * @return the underlying object of this {@link ObjectImpl}
+     *
+     * @throws KException if an error occurs
+     */
+    public Node node(UnitOfWork uow) throws KException {
```

Comment:
In that case and sounds like it should not be public, I can call it reflectively from the test framework.

I did use the get() method initially but this does not work since the method assumes nodes are under the path /tko:komodo/tko:workspace so fails for / and /tko.komodo. Given this is the case, should the method be renamed to getFromWorkspace() since I can see this cropping up as a common confusion?